### ISSUE TITLE
string.IndexOfAny is implemented incorrectly

### ISSUE THREAD

**Issue by athendrix** [bridgedotnet/Bridge#3964]
A description of the issue.
`string.IndexOfAny()` is implemented incorrectly.
The order of the input character array affects the outcome and it shouldn't.
For example, if you switch `H` and `d` in the character array in the example below, you'll notice that the example outputs `0` as expected instead of `11`.

### Steps To Reproduce

https://deck.net/37dbfd6bb322075afc63a3a31b8366d2

https://dotnetfiddle.net/w8cu4N

```csharp
public class Program
{
    public static void Main()
    {
        char[] test = {'d','H'};
        var msg = "Hello, World!";
        
        Console.WriteLine(msg.IndexOfAny(test));
    }
}
```

### Expected Result

The console should output `0` here.

### Actual Result

The console outputs `11`.

**Comment by geoffreymcgill** [comment]
Thanks for the report. We will investigate and try to fix in time for the upcoming 17.8.0 release.

**Comment by geoffreymcgill** [comment]
A fix for this issue has been submitted. Once the fix has been reviewed and the second round of testing is verified, the fix will be included in the upcoming Bridge 17.8.0 release.

**Comment by fabriciomurta** [comment]
The pull request for this issue looks good and will make it to the next Bridge release!
